Send All.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE MudletPackage>
+<MudletPackage version="1.001">
+    <TriggerPackage/>
+    <TimerPackage/>
+    <AliasPackage>
+        <Alias isActive="yes" isFolder="no">
+            <name>Global Command</name>
+            <script>send(matches[2])
+raiseGlobalEvent(&quot;my_event&quot;, matches[2])
+</script>
+            <command></command>
+            <packageName></packageName>
+            <regex>^:(.*)$</regex>
```

Comment:
Forgive me, I am a little rusty on regexps, but what is that `:` doing in `^:(.*)$` - it seems that it makes a leading colon a requirement - Oh 💡 , that is the "trigger" i.e. "begin your command with a colon to send the command from all active profiles...